maintainers/release-process.md

```diff
@@ -194,8 +194,54 @@ release:
 
 * Bump the version number of the release branch as above (e.g. to
   `2.12.2`).
-  
+
 ## Recovering from mistakes
 
 `upload-release.pl` should be idempotent. For instance a wrong `IS_LATEST` value can be fixed that way, by running the script on the actual latest release.
 
+## Security releases
```

Comment:
```suggestion
## Security reports handling and release
```

I'm taking the liberty to broaden the scope, because we have a clear need for it. It should probably be in its own page instead of `release-process.md`.